exercises/J-functions/exercise2.js

```diff
@@ -1,7 +1,11 @@
 function triple(number) {
   // complete function here
+  var number = 12;
```

Comment:
I think you don't need write step 3  (var number = 12;)  as passed "number" as parameter in the function.